packages/forms/src/Components/Select.php

```diff
@@ -988,6 +988,42 @@ static function (Select $component): bool {
         $this->saveRelationshipsUsing(static function (Select $component, Model $record, $state) use ($modifyQueryUsing) {
             $relationship = $component->getRelationship();
 
+            if ($relationship instanceof HasOne || $relationship instanceof HasMany) {
```

Comment:
Maybe check `HasOneOrMany` instead of both?

Reply:
That would then also include the morph relation types (MorphOne, MorphMany) which would fail with the added logic

Reply:
Ah okay